### ISSUE TITLE
Transcode music: Add a "Clear" button to the "Details..." window

### ISSUE THREAD

**Issue by Clementine-Issue-Importer** [clementine-player/Clementine#3083]
_From [mghansen256](https://code.google.com/u/mghansen256/) on July 29, 2012 22:12:11_

The transcoding log shown by the "Details..." button can currently not be cleared, and even persists after closing and re-opening the dialog. Would be nice to have a clear button, to be used before starting a new encoding.

_Original issue: http://code.google.com/p/clementine-player/issues/detail?id=3082_
